x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_integrate.tsx

```diff
@@ -26,86 +29,452 @@ interface AnalyticsCollectionIntegrateProps {
         {
-          defaultMessage: 'DNS URL',
+          defaultMessage: 'Javascript Client',
+        }
+      ),
+    },
+    {
+      key: 'searchuiEmbed',
+      title: i18n.translate(
+        'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.searchuiEmbed.title',
+        {
+          defaultMessage: 'Search UI',
         }
       ),
-      description: analyticsDNSUrl,
     },
   ];
-  const webclientSrc = getEnterpriseSearchUrl('/analytics.js');
 
   return (
-    <>
-      <EuiTitle>
-        <h4>
+    <EuiPanel hasBorder paddingSize="l">
+      <EuiTitle size="s">
+        <h2>
           {i18n.translate(
-            'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.credentials.headingTitle',
-            {
-              defaultMessage: 'Credentials',
-            }
-          )}
-        </h4>
-      </EuiTitle>
-      <EuiSpacer size="s" />
-      <EuiPanel hasShadow={false} color="subdued" paddingSize="xl" grow={false}>
-        <EuiDescriptionList listItems={credentials} type="column" align="center" />
-      </EuiPanel>
-
-      <EuiSpacer size="l" />
-
-      <EuiTitle>
-        <h4>
-          {i18n.translate(
-            'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.embed.headingTitle',
+            'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.title',
             {
               defaultMessage: 'Start tracking events',
             }
           )}
-        </h4>
+        </h2>
       </EuiTitle>
-      <EuiSpacer size="s" />
-      <EuiText size="s">
-        <p>
-          {i18n.translate(
-            'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.embed.description',
-            {
-              defaultMessage:
-                'Embed the JS snippet below on every page of the website or application you’d like to tracks.',
-            }
-          )}
-        </p>
-      </EuiText>
-      <EuiSpacer size="s" />
-      <EuiCodeBlock language="html" isCopyable>
-        {`<script src="${webclientSrc}" data-dsn="${analyticsDNSUrl}" defer></script>`}
-      </EuiCodeBlock>
-
-      <EuiSpacer size="l" />
-      <EuiText size="s">
-        <FormattedMessage
-          id="xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.scriptDescription"
-          defaultMessage="Track individual events, like clicks, by calling the {trackEvent} method."
-          values={{
-            trackEvent: <strong>trackEvent</strong>,
-          }}
-        />
-      </EuiText>
-      <EuiSpacer size="s" />
-      <EuiCodeBlock language="js" isCopyable>
-        {`window.elasticAnalytics.trackEvent("click", {
-  title: "Website Analytics",
-  url: "www.elastic.co/analytics/overview"
-})`}
-      </EuiCodeBlock>
-    </>
+      <EuiSpacer size="m" />
+      <EuiTabs>
+        {tabs.map((tab) => (
+          <EuiTab
```

Comment:
Would be good to add a data-telemetry-id to the tab selector for tracking purposes